recipes/prismatic/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set version = "1.2.1" %}
+
+package:
+  name: prismatic
+  version: {{ version }}
+
+source:
+  url: https://github.com/prism-em/prismatic/archive/{{ version }}.tar.gz
+  sha256: 844f659575dd98999b7c4864e850a7ec735f297ab1fb6c2d2be91d7f9720fe70 
+  patches:
+    - cmakelists.patch
+
+build:
+  number: 0
+  # will skip windows & linux for the moment - until we've got OSX sorted...
+  skip: True             # [win]
+  skip: False            # [linux]
+  skip: False            # [osx]
+  features:
+
+requirements:
+  build:
+    # If your project compiles code (such as a C extension) then add the required compilers as separate entries here.
+    # Compilers are named 'c', 'cxx' and 'fortran'.
+    - {{ compiler('c') }}                
+    - {{ compiler('cxx') }}              
+    - {{ cdt('libxext-devel') }}         # [linux]
+    - {{ cdt('mesa-libgl-devel') }}      # [linux]
+    - {{ cdt('mesa-dri-drivers') }}      # [linux]
+    - {{ cdt('libxau-devel') }}          # [linux]
+    - {{ cdt('libselinux-devel') }}      # [linux]
+    - {{ cdt('libxdamage') }}            # [linux]
+    - {{ cdt('libxfixes') }}             # [linux]
+    - {{ cdt('libxxf86vm') }}            # [linux]
+    - make                               # [unix]
+    - cmake
+  host:
+    #- python >=3
+    #- pip
+    - boost
+    - hdf5
+    - fftw
+    - qt =5.9.*
+    #- numpy >=1.13.0
+    #- matplotlib >=2.0.2
+    - h5py
+    - zlib
+  run:
+    - fftw
+    - libcxx  # [osx]
```

Comment:
```suggestion
```

Reply:
I added libcxx to the run requirements because the OSX tests were failing otherwise (even though the build was successful). Is there an alternative to including libcxx here that you would recommend?

Reply:
When you include the compilers it should add it implicit